src/CoreWCF.UnixDomainSocket/src/CoreWCF/Channels/UnixPosixIdentityBindingElement.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net.Security;
+using CoreWCF.Security;
+
+namespace CoreWCF.Channels
+{
+    public class UnixPosixIdentityBindingElement : StreamUpgradeBindingElement
+    {
+        private ProtectionLevel _protectionLevel;
+
+        public UnixPosixIdentityBindingElement()
+            : base()
+        {
+            _protectionLevel = ConnectionOrientedTransportDefaults.ProtectionLevel;
+        }
+
+        protected UnixPosixIdentityBindingElement(UnixPosixIdentityBindingElement elementToBeCloned)
+            : base(elementToBeCloned)
+        {
+            _protectionLevel = elementToBeCloned._protectionLevel;
+        }
+
+        public ProtectionLevel ProtectionLevel
+        {
```

Comment:
ProtectionLevel doesn't mean anything in the context of this class. It doesn't provide encryption or signing of the data.